src/images.py

```diff
@@ -0,0 +1,177 @@
+"""Module for matching individual images from raw and corrected directories
+and computing intensity slopes from the foreground and background regions."""
+import warnings
+
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+def match_image_pairs(raw_dir, corrected_dir, key_parts=4, extensions=None):
```

Comment:
How might someone use these functions through a CLI? Consider brainstorming on this for future iterations.